modules/nf-core/ilastik/multicut/main.nf

```diff
@@ -0,0 +1,38 @@
+process ILASTIK_MULTICUT {
+    tag "$meta.id"
+    label 'process_low'
+
+    container "labsyspharm/mcmicro-ilastik:1.6.0"
```

Comment:
Can this be added to conda?

Reply:
So it's a bit tricky with ilastik. It is already on conda, but on it's own channel called [ilastik-forge](https://anaconda.org/ilastik-forge). There is also a biocontainer, but I don't see from where it is built : https://hub.docker.com/r/biocontainers/ilastik/tags. Additionally, the biocontainer is not the most recent release candidate. This is problematic, since MacOS users can't use the latest stable release to train models but can use the release candidate. That's why for now I put our mcmicro container, which we can update and uses the latest release.

![image](https://user-images.githubusercontent.com/11639154/220347868-735415e5-24d7-431f-a751-b3f892a2ef8b.png)

Reply:
Wow I've never seen a tool have it's own specific conda channel. Is it impossible to use this channel in Nextflow?

Reply:
I actually have no idea. @drpatelh is a custom conda channel like ilastik-forge useable in the conda definition of modules?

Reply:
I am gonna check with the ilastik developers, whether they plan to have a maintained container on dockerhub that tracks the newest version. In the meantime, I think having the ilastik modules run off the latest mcmicro-ilastik container is the best way to go. That way we can update the container whenever we need to.

Reply:
Allright, can you also ask whether or not they would want to add their tool to `biocontainers` instead of that separate channel? I think it would be best for everyone (although I don't know what their reason is so we'll have to see)